src/redux/reducer.js

```diff
@@ -182,9 +182,22 @@ function reducer(state = initialState, action) {
         highlightedFeature: action.feature,
       });
     case Constants.ADD_COMPARISON_FEATURE:
-      return {...state, comparisonFeatures: state.comparisonFeatures.concat([action.feature])};
+    {
+      let collapsibleState = state.collapsibleState;
+      collapsibleState['Locations to Compare'] = true;
```

Comment:
Is some cases this will not work because `...state.collapsibleState` will override the `true`. I have to initialize `collapsibleState` with a empty object `{}`.

Reply:
Ah, you're right. It would have to be flipped around to `let collapsibleState = {...state.collapsibleState, ['Locations to Compare']: true};` for "Locations to Compare" to override any existing value.